src/components/inspectors/TimerExpression.vue

```diff
@@ -212,7 +212,7 @@ export default {
       this.update();
     },
   },
-  mounted() {
```

Comment:
The reason this works is because `created` is run before this component mounts, and hence before the date picker child component mounts. What was happening before is the date picker component would mount, then emit an event right away to set the default date, since at that point `parseTimerConfig` hasn't run and so the date picker did not yet receive a value. This would overwrite the end date set during the parsing of the timer config.

Running `parseTimerConfig` before the date picker mounts ensures that the end date is set, and so the date picker would mount with an actual value and not emit its default value.